config/locales/validations/sales/setup/en.yml

```diff
@@ -0,0 +1,25 @@
+en:
+    validations:
+        sales:
+            setup:
+                saledate:
+                    not_within:
+                        next_two_weeks: "Sale completion date must not be later than 14 days from today’s date."
+                    must_be_within:
+                        current_collection_year:
+                            "Enter a date within the %{current_start_year_short}/%{current_end_year_short} collection year, which is between %{current_start_year_long} and %{current_end_year_long}."
+                        previous_and_current_collection_year:
+                            "Enter a date within the %{previous_start_year_short}/%{previous_end_year_short} or %{previous_end_year_short}/%{current_end_year_short} collection years, which is between %{previous_start_year_long} and %{current_end_year_long}."
+                    invalid:
+                        merged_organisations:
+                            "Enter a date when the owning organisation was active. %{owning_organisation} became inactive on %{merge_date} and was replaced by %{absorbing_organisation}."
+                        absorbing_organisations:
+                            "Enter a date when the owning organisation was active. %{owning_organisation} became active on %{available_from}."
+
+                owning_organisation:
+                    inactive:
+                        merged_organisation:
+                            "The owning organisation must be active on the sale completion date. %{owning_organisation} became inactive on %{merge_date} and was replaced by %{absorbing_organisation}."
+                        absorbing_organisation:
+                            "The owning organisation must be active on the sale completion date. %{owning_organisation} became active on %{available_from}."
+
```

Comment:
would it make sense for this file to be setup.en.yml instead of being in a setup folder?

Reply:
Also, the formatting looks a bit off (?) is it 2 tabs for each nesting? Could you check other yml files to make sure the spacing is correct?

Reply:
Yes will change that